lib/checkPullRequestBranch.js

```diff
@@ -23,28 +22,62 @@ const checkBranch = async (context) => {
    */
   const pullRequest = context.payload.pull_request;
   const branchName = pullRequest.head.ref;
-  const branchIsInvalid = DISSALLOWED_BRANCH_PREFIXES.some((prefix) => {
-    return branchName.startsWith(prefix);
-  });
 
-  if (branchIsInvalid) {
+  message = '';
+  invalid_branch_check = false;
+  
+
+  if(branchName === 'develop'){
+    
+    invalid_branch_check = true;
+    message = ', PRs made from develop branch are not allowed. Also branches '+
+              'whose name is prefixed with develop, release or test are not allowed.';
+  }
```

Comment:
```suggestion
  if (branchName === 'develop') {
    invalid_branch_check = true;
    message = (
      ', PRs made from develop branch are not allowed. Also branches ' +
      'whose name is prefixed with develop, release or test are not allowed.'
    );
  }
```

Reply:
@vojtechjelinek Is this still required, as the message size is reduced??